packages/fether-react/src/App/style.js

```diff
@@ -0,0 +1,24 @@
+// Copyright 2015-2018 Parity Technologies (UK) Ltd.
+// This file is part of Parity.
+//
+// SPDX-License-Identifier: BSD-3-Clause
+
+import { css } from 'styled-components';
+
+const DivContentStyles = css`
+  margin: 0 auto;
+  max-width: 22rem;
+  overflow: visible;
+`;
+
+const DivWindowStyles = css`
```

Comment:
I'm new to styled-components so I'm not sure what the best practices are, but from what I understood, we don't manage styles anymore, we only manage components.

So I would actually not use css\`\` here, but styled.div\`\`, and remove all css\`\`. Following styled-components.com, the only situation I would use css\`\` is when you have a special styling case for a component, for example depending on some props. Everything else is a React component.

Reply:
One convention that we're experimenting in substrate light ui, that we could use here is:
- next to MyComponent.js, create MyComponent.styles.js
- if MyComponent.js needs some styled div, span other other components, put them in MyComponent.styles.js

What do you think @ltfschoen?

Reply:
Yes, I originally only adopted css`` by accident. I like the way we're doing it in substrate-light-ui and I've pushed commits to use MyComponent.styles.js as you suggested.
I still think using css`` is useful to keep the contents of MyComponent.styles.js DRY. For example if we had a MyComponent.styles.js that contained multiple components where we wanted to apply the same animation, we'd have to add ~10 lines of `animation-` properties to each component, whereas we could define say const animationRuleSlideLeftIn = css`...` and then apply it to each component with just `${animationRuleSlideLeftIn};`